mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -45,6 +46,9 @@ private[ml] trait CrossValidatorParams extends ValidatorParams {
   def getNumFolds: Int = $(numFolds)
 
   setDefault(numFolds -> 3)
+
+  /** @group setParam */
+  def setSeed(value: Long): this.type = set(seed, value)
```

Comment:
This needs to be in the CrossValidator class itself, not the trait (for Java compatibility)